applications/CompressiblePotentialFlowApplication/custom_elements/compressible_potential_flow_element.h

```diff
@@ -83,678 +74,223 @@ class CompressiblePotentialFlowElement : public Element
     ///@name Life Cycle
     ///@{
 
+    // Constructors.
+
+    /// Default constuctor.
     /**
-     * Constructor.
+     * @param NewId Index number of the new element (optional)
      */
-    explicit CompressiblePotentialFlowElement(IndexType NewId = 0) {
-        KRATOS_WARNING_ONCE("DEPRECATED") << "CompressiblePotentialFlowElement was replaced by "
-            << "IncompressiblePotentialFlowElement, it will be removed soon!" << std::endl;
-    };
+    explicit CompressiblePotentialFlowElement(IndexType NewId = 0){}
 
     /**
      * Constructor using an array of nodes
      */
-    CompressiblePotentialFlowElement(IndexType NewId, const NodesArrayType& ThisNodes):Element(NewId, ThisNodes) {
-        KRATOS_WARNING_ONCE("DEPRECATED") << "CompressiblePotentialFlowElement was replaced by "
-            << "IncompressiblePotentialFlowElement, it will be removed soon!" << std::endl;
-    };
+    CompressiblePotentialFlowElement(IndexType NewId, const NodesArrayType& ThisNodes)
+        : Element(NewId, ThisNodes){}
 
     /**
      * Constructor using Geometry
      */
-    CompressiblePotentialFlowElement(IndexType NewId, GeometryType::Pointer pGeometry):Element(NewId, pGeometry) {
-        KRATOS_WARNING_ONCE("DEPRECATED") << "CompressiblePotentialFlowElement was replaced by "
-            << "IncompressiblePotentialFlowElement, it will be removed soon!" << std::endl;
-    };
+    CompressiblePotentialFlowElement(IndexType NewId, GeometryType::Pointer pGeometry)
+        : Element(NewId, pGeometry){}
 
     /**
      * Constructor using Properties
      */
-    CompressiblePotentialFlowElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties):Element(NewId, pGeometry, pProperties) {
-        KRATOS_WARNING_ONCE("DEPRECATED") << "CompressiblePotentialFlowElement was replaced by "
-            << "IncompressiblePotentialFlowElement, it will be removed soon!" << std::endl;
-    };
+    CompressiblePotentialFlowElement(IndexType NewId,
+                                       GeometryType::Pointer pGeometry,
+                                       PropertiesType::Pointer pProperties)
+        : Element(NewId, pGeometry, pProperties){}
 
     /**
      * Copy Constructor
      */
-    CompressiblePotentialFlowElement(CompressiblePotentialFlowElement const& rOther) {};
+    CompressiblePotentialFlowElement(CompressiblePotentialFlowElement const& rOther) = delete;
```

Comment:
If you delete the copy constructor, you should also delete the assignment operator. Or define both of them. Same for move operations.

Reply:
I solved this here https://github.com/KratosMultiphysics/Kratos/pull/4642/commits/76a0a5f4fd9d5f3944c2f304ad7620534b167b54 and here https://github.com/KratosMultiphysics/Kratos/pull/4679